### PR TITLE
Added russian translations

### PR DESCRIPTION
**Pull request by buslov**

### CODE REVIEW

**Comment by RichardLitt** [comment]
Thanks @buslov. Is this related to #154, or a new effort?

**Comment by buslov** [comment]
> Thanks @buslov. Is this related to #154, or a new effort?

It's a new effort. Not related to #154.

**Comment by RichardLitt** [comment]
Thanks. @qWici, what do you think?

File: README_ru.md [hunk 1]
```diff
@@ -0,0 +1,1272 @@
...
+* [GitUp](https://github.com/git-up/GitUp) - новый графический клиент, имеющий весьма своеобразные методы работы со сложностями git
+* [gitx-dev](https://rowanj.github.io/gitx/) - еще один графический git-клиент для OS X
+* [Source Tree](https://www.sourcetreeapp.com/) - свободный графический git-клиент для Windows и OS X
+* [Tower](http://www.git-tower.com/) - графический git-клиент для OS X (платный)
+* [tig](https://jonas.github.io/tig/) - консольный текстовый интерфейс для Git
+* [Magit](https://magit.vc/) - интерфейс для git, реализованный в виде модуля Emacs.
+* [GitExtensions](https://github.com/gitextensions/gitextensions) - расширение оболочки, плагин для Visual Studio 2010-2015 и автономный инструмент для управления репозиториями Git.
+* [Fork](https://git-fork.com/) - быстрый и дружелюбный git-клиент для Mac (бета)
+* [gmaster](https://gmaster.io/) - git-клиент для Windows с трехсторонним слиянием, обнаружением рефакторинга, семантическим сравнением и слиянием (бета)
+* [gitk](https://git-scm.com/docs/gitk) - git-клиент под Linux для для просмотра состояния репозитория.
```
Comment (qWici) [inline README_ru.md]: *..под Linux **для** просмотра..*

**Comment by qWici** [comment]
@RichardLitt u can close my #154 PR, and after correcting the errors about which I mentioned above merge this PR

@buslov отличная работа!

**Comment by buslov** [comment]
@RichardLitt I have fixed the typos that @qWici found.

@qWici спасибо за проверку!

**Comment by RichardLitt** [comment]
In that case, thank you both! We can also add you as collaborators if you're interested in keeping the translations up to date.

**Comment by qWici** [comment]
@RichardLitt Of course. I will maintain russian translation and update it when updating the original

**Comment by buslov** [comment]
@RichardLitt OK. It's cool!

**Pull request merged**